src/Source.jl

```diff
@@ -139,6 +139,7 @@ function Source(dsn::DSN, query::AbstractString; weakrefstrings::Bool=true, noqu
         allocsize = rowset = rows
     else
         rowset = allocsize = 1
+        rows = Missings.missing  # -1 is an invalid number for Data.Schema rows
```

Comment:
Better to just have `using Missings: missing` at the top of the file since `missing` is included in Base in 0.7

Reply:
@omus So when I include that at the top of the file and set `rows = missing` I get the error:
```
 MethodError: no method matching DataStreams.Data.Schema(::Array{Type,1}, ::Array{String,1}, ::Base.RefValue{Int16}, ::Dict{String,Any})
  Closest candidates are:
    DataStreams.Data.Schema(::Any, ::Any, ::Union{Integer, Missings.Missing}, ::Dict) at /root/.julia/v0.6/DataStreams/src/DataStreams.jl:63
```

Reply:
I'll take a look

Reply:
Looks like `using Missings` happens in "src/ODBC.jl" already.